rbac/rbac/middleware.py

```diff
@@ -253,10 +250,8 @@ def process_request(self, request):  # noqa: C901
                                                              request)
 
             with tenant_context(tenant):
-                try:
-                    Principal.objects.get(username__iexact=username)
-                except Principal.DoesNotExist:
-                    Principal.objects.create(username=username)
+                created = Principal.objects.get_or_create(username=username)[1]
```

Comment:
It might be more clear to do something like this?
```suggestion
                _, created = Principal.objects.get_or_create(username=username)
```

Reply:
👍